wpimath/src/main/java/edu/wpi/first/math/geometry/Rotation2d.java

```diff
@@ -153,6 +153,24 @@ public Rotation2d rotateBy(Rotation2d other) {
         m_cos * other.m_cos - m_sin * other.m_sin, m_cos * other.m_sin + m_sin * other.m_cos);
   }
 
+  /**
+   * Returns the absolute value of the Rotation2d.
+   *
+   * @return The new absolute value Rotation2d
+   */
+  public Rotation2d abs() {
+    return new Rotation2d(Math.abs(m_value));
+  }
+
+  /**
+   * Flips the Rotation2d to the opposite direction.
+   *
+   * @return The new flipped Rotation2d
+   */
+  public Rotation2d flip() {
```

Comment:
Why would users want to do this? Is it that common that users should call this instead of just calling rotateBy() themselves?